oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/LastRevRecoveryAgent.java

```diff
@@ -310,6 +310,12 @@ public void sweepUpdate(Map<Path, UpdateOp> updates)
         long lastLog = startOfScan;
 
         for (NodeDocument doc : suspects) {
+        	
+        	//Should skip previous documents
+        	if(!doc.getPath().isAbsolute()) {
```

Comment:
I would rather make a decision on NodeDocument.getSplitDocType(). WDYT?

Reply:
Yes that is better, I will use NodeDocument.isSplitDocument() to check previous/split document

Reply:
This is even better. +1